runtime/arc.js

```diff
@@ -102,11 +102,12 @@ class Arc {
     return [...this._viewsById.values()];
   }
 
-  clone() {
+  // Makes a copy of the arc used for speculative execution.
+  makeSpeculative() {
```

Comment:
It can't return the particle because of the PEC barrier, so renaming might be better. OTOH, this *is* instantiating a particle .. just on the other side of the barrier.